model/textparse/openmetricsparse.go

```diff
@@ -94,16 +94,51 @@ type OpenMetricsParser struct {
 	exemplarVal   float64
 	exemplarTs    int64
 	hasExemplarTs bool
+
+	// lines ending with _created are skipped by default as they are
+	// parsed by the CreatedTimestamp method. The skipCT flag is set to
+	// false when the CreatedTimestamp method is called.
+	skipCT bool
+}
+type openMetricsParserOptions struct {
+	// SkipCT skips the parsing of _created lines by default.
```

Comment:
```suggestion
	// SkipCT skips the parsing of _created lines.
```

We are changing default here, no?

Reply:
applied this